### PR TITLE
[UPD] apriori: account_bank_statement_import > account_statement_import

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
You can merge when ready

**Comment by pedrobaeza** [comment]
Please don't accumulate several of this in the same PR. Each one should be a PR.

**Comment by MiquelRForgeFlow** [comment]
...ok, I will merge then.

**Comment by pedrobaeza** [comment]
Hey, you have merged with all the changes... I meant to only put the first one.

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -40,6 +62,7 @@
     "pos_reprint": "point_of_sale",
     "website_theme_install": "website",
     # OCA/partner-contact
+    "base_vat_sanitized": "base_vat",
```
Comment (pedrobaeza) [inline openupgrade_scripts/apriori.py:65]: I don't have clear this one.

**Pull request merged**